lib/util/Components.js

```diff
@@ -99,9 +99,12 @@ Components.prototype.list = function() {
       component = this.get(node);
     }
     if (component) {
-      usedPropTypes[this._getId(component.node)] = (this._list[i].usedPropTypes || []).filter(function(propType) {
+      var newUsedProps = (this._list[i].usedPropTypes || []).filter(function(propType) {
```

Comment:
I agree; we should only use `const` and `let`.

Regardless, we shouldn't add new instances of `var` - a mix is preferable to that.

I'd prefer manual modification to a codemod.